src/v2/view-builder/views/TerminalView.js

```diff
@@ -1,14 +1,24 @@
-import { createCallout } from 'okta';
+import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals/BaseFooter';
+import {getBackToSignInLink} from '../utils/LinksUtil';
 
 const Body = BaseForm.extend({
   noButtonBar: true,
+
   postRender () {
     BaseForm.prototype.postRender.apply(this, arguments);
     this.$el.addClass('terminal-state');
   },
 
+  title () {
+    if (this.options.appState.isExpiredEmailLinkTerminalState()) {
+      return loc('oie.email.return.link.expired.title', 'login');
+    }
+    return BaseForm.prototype.title();
```

Comment:
`BaseForm.prototype.title.apply(this, arguments);`

Reply:
will change, but I'm curious why thats better to use?

Reply:
`BaseForm.prototype.title` may mess up `this` context.

Reply:
Basically calling the base method we are ignoring the current context of this view, If we had some options that needed to be propagated they won't go if we don't apply the context..